ams/server/src/main/java/com/netease/arctic/server/IcebergRestCatalogService.java

```diff
@@ -88,14 +99,13 @@ public class IcebergRestCatalogService extends PersistentBase {
       Collections.unmodifiableSet(Sets.newHashSet(CatalogMetaProperties.KEY_WAREHOUSE));
 
   private final JavalinJackson jsonMapper;
-  private final ObjectMapper objectMapper;
 
   private final TableService tableService;
   private final MetricsManager metricsManager;
 
   public IcebergRestCatalogService(TableService tableService) {
     this.tableService = tableService;
-    this.objectMapper = jsonMapper();
+    ObjectMapper objectMapper = jsonMapper();
```

Comment:
Should we consider changing the name of this class, as it can now handle requests for both Iceberg Format and Mixed-Iceberg Format tables simultaneously?